pulp_certguard/app/__init__.py

```diff
@@ -0,0 +1,6 @@
+from pulpcore.plugin import PulpPluginAppConfig
+
+
+class PulpCertGuardPluginAppConfig(PulpPluginAppConfig):
+    name = 'pulp_certguard.app'
```

Comment:
Do we want to adopt the convention for s/pulp_certguard/certguard/ similar to the other plugins? I think I'm maybe +0? What do you think?

Reply:
Yes.

Reply:
Can we remove `pulp_` in the label as well?